assets/src/design-system/components/input/stories/index.js

```diff
@@ -28,6 +28,7 @@ import { text } from '@storybook/addon-knobs';
 import { Input } from '..';
 import { DarkThemeProvider } from '../../../storybookUtils';
 import { Headline } from '../../..';
+import { Close, LetterSpacing } from '../../../icons';
```

Comment:
Please update from `main` to get the newest icon definitions. These are correctly scaled and meant to only ever be shown at 32x32 px sizing. Also there's been some slight renaming.